packages/atomic/src/components/atomic-result-placeholder/atomic-result-placeholder.tsx

```diff
@@ -20,7 +20,7 @@ const placeholderClasses = 'block bg-neutral w-full h-full rounded';
 export class AtomicResultPlaceholder {
   @Prop() display!: ResultDisplayLayout;
   @Prop() density!: ResultDisplayDensity;
-  @Prop() imageSize!: ResultDisplayImageSize;
+  @Prop() imageSize?: ResultDisplayImageSize;
```

Comment:
Why make it optional now? or maybe why aren't they all optional as they have no default assigned values